fixity/fixity.py

```diff
@@ -233,19 +248,21 @@ def scanall(
                 report_auth=report_auth,
                 session_id=session_id,
                 force_local=force_local,
+                timestamp=timestamp,
             )
             if not scan_success:
                 success = False
         except Exception as e:
             utils.pyprint(
                 f"Internal error encountered while scanning AIP {aip['uuid']} ({type(e).__name__})",
                 file=sys.stdout,
+                display_time=timestamp,
```

Comment:
Notice how the same argument is passed to `scanall` as `timestamp` but `pyprint` receives it as `display_time`. Since it represents the same concept (a boolean that indicates if timestamps will be added to each line of the output) don't you think it should be name consistently across the whole code base?

Reply:
ok, sounds good. Let me change it